rest-api/tools/import_organizations.py

```diff
@@ -181,12 +180,16 @@ def _entity_from_row(self, row):
                 link=link)
 
   def _get_lat_long_for_site(self, address_1, city, state):
-    api_key = os.environ.get('API_KEY')
-    gmaps = googlemaps.Client(key=api_key)
-    geocode_result = gmaps.geocode(address_1 + '' +  city + ' ' +  state)[0]
-    latitude = geocode_result['geometry']['location']['lat']
-    longitude = geocode_result['geometry']['location']['lng']
-    return latitude, longitude
+    try:
+        api_key = os.environ.get('API_KEY')
+        gmaps = googlemaps.Client(key=api_key)
+        geocode_result = gmaps.geocode(address_1 + '' +  city + ' ' +  state)[0]
```

Comment:
Do we not care about address 2 for geocoding purposes?

Does there need to be a comma between city and state, or does it not care?

Reply:
Ping?

Reply:
No, what the Google API expects is a human readable string (without any additional info such as suite, bldg., room, etc.) i.e. "1600 Amphitheatre Parkway Mountain View CA 94043" 
and the postal_code is optional.